tests/components/forked_daapd/test_media_player.py

```diff
@@ -710,6 +710,9 @@ async def test_librespot_java_stuff(
     await hass.async_block_till_done()
     state = hass.states.get(TEST_MASTER_ENTITY_NAME)
     assert state.attributes[ATTR_INPUT_SOURCE] == "librespot-java (pipe)"
+    # test title and album not reversed when data_kind not url
+    assert state.attributes[ATTR_MEDIA_TITLE] == "librespot-java"
+    assert state.attributes[ATTR_MEDIA_ALBUM_NAME] == "some album"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5633381)